doc/nrf/known_issues.rst

```diff
@@ -1847,7 +1848,7 @@ KRKNWK-12482: Reception of correct frames can occasionally end in failure with e
 KRKNWK-11384: Assertion with Bluetooth® LE and multiprotocol usage
   The device might assert on rare occasions during the use of Bluetooth® LE and 802.15.4 multiprotocol.
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
 
 KRKNWK-11204:
```

Comment:
This is the limitation of 802.15.4 Radio Driver, not a known-issue. This shall be moved to the Limitations.

Reply:
Addressed in https://github.com/nrfconnect/sdk-nrfxlib/pull/809